documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md

```diff
@@ -0,0 +1,15 @@
+## Meaningful data
+
+<!-- Alt title: Towards semantic interoperability: the role of open standardization -->
+
+Clarity about the meaning of data is essential to ensure that data can be accurately and consistently interpreted and used by different people and systems. When the meaning of shared data is unclear, it can lead to miscommunication and misinterpretation, resulting in errors and poor decisions. Organizations that find themselves dealing with ambiguous data therefore spend a lot of effort in mapping that data to the formats and structures that their IT systems expect. Since this is time-consuming and costly, the lack of shared meaning of data is a major barrier to data sharing and therefore to the realisation of the Digital Single Market strategy.
+
+The ability of IT systems to exchange data with unambiguous, shared meaning is called semantic interoperability. Semantic interoperability is an essential requirement for federated data networks such as IDS. It requires that data providers and data consumers in the network express their data offering or need using explicit reference to a common vocabulary. The commonality is important; if data is provided with references to a vocabulary that the receiving party is unfamiliar with, then the need to spend integration effort on their end remains. Vocabularies become common (shared) through the process of standardization.
+
+Many industries and other business ecosystems have turned to open standardization as a means to achieve semantic interoperability between its members. Open standardization means that members collaboratively maintain and develop semantic standards. It is a continuous balancing act between the need for strict uniformity to keep data consistent and easy to understand, and the need to accomodate for the fact that different organizations have different requirements for their data. 
+
+This means there is often a limit to the level of semantic interoperability that can be achieved. Every member in a business ecosystem operates with a different world view. These differences arise from operating in different jurisdictions, in different domains, carrying out different business processes, serving different marktes, offering different services, and so on. High variety between members in business ecosystems means that any semantic standard for that community will have to allow for flexibility, which means some integration effort will remain necessary. Low variety allow for stricter semantic standards that bring more uniformity, allowing for more efficient data sharing and automatisation.  
+
+In any case, governance has to be put in place to make sure that a semantic standard serves the needs of the community as best as possible and will remain doing so. How these governance processes can be organized is discussed in section 4.  
```

Comment:
```suggestion
In any case, governance has to be put in place to make sure that a semantic standard serves the needs of the community as best as possible and will remain doing so. How these governance processes can be organized is discussed in section 4.
```
Two questions:
* "as best as possible" doesn't work. The proper adverb here would be "well". So, can one say "serves the needs of the community as _well_ as possible", or would that result in the "as well as" being mis-read as "and"? In that case, would we have an alternative idea how to phrase it?
* Do we have to write section numbers, or can we refer to other sections by ID?

Reply:
as good as possible